chapter_11_practic/task3.cpp

```diff
@@ -0,0 +1,118 @@
+// As we saw earlier, it�s possible to make a class that acts like an array.The CLARRAY
+// example shown here is a complete program that shows one way to create your own array
+// class :
+// The output of this program is
+// 0 1 4 9 16 25 36 49 64 81
+// Starting with CLARRAY, add an overloaded assignment operator and an overloaded copy
+// constructor to the Array class.Then add statements such as
+// Array arr2(arr1);
+// and
+// arr3 = arr1;
+// to the main() program to test whether these overloaded operators work.The copy constructor should create an entirely new Array object with its own memory for storing
+// array elements.Both the copy constructorand the assignment operator should copy the
+// contents of the old Array object to the new one.What happens if you assign an Array of
+// one size to an Array of a different size ?
+
+#include <iostream>
+
+class Array
+{
+private:
+	int* ptr;
+	int size;
+public:
+	Array() : ptr(0), size(0)
+	{ }
+	Array(int s)
+	{
+		size = s;
+		ptr = new int[s];
+	}
+	Array(Array& arr)
```

Comment:
the signature should have a constant reference

Reply:
fixed in https://github.com/Dmitriy-Romanchuk/master/pull/11/commits/333b9e5d259dc07a465078055cb8bd5909b6599b